.github/workflows/ci.yml

```diff
@@ -0,0 +1,67 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+
+env:
+  RUBY_VERSION: 2.7.2
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        java-version:
+          - 8
+          - 11
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Validate Gradle Wrapper
+        uses: gradle/wrapper-validation-action@v1
+
+      - name: Prepare Cache
+        run: |
+          rm -f  ~/.gradle/caches/modules-2/modules-2.lock
+          rm -fr ~/.gradle/caches/*/plugin-resolution/
+
+      - name: Cache Gradle Files
+        uses: actions/cache@v3
+        with:
+          path:  |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: v1-${{ runner.os }}-gradle-${{ matrix.java-version }}-${{ hashFiles('**/*.gradle', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            v1-${{ runner.os }}-gradle-${{ matrix.java-version }}-
+
+      - name: Configure Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ env.RUBY_VERSION }}
+          bundler-cache: true
+
+      - name: Configure JDK
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: ${{ matrix.java-version }}
+
+      - name: Connected Test
+        uses: reactivecircus/android-emulator-runner@v2
+        with:
+          api-level: 22
+          emulator-options: --force -n test -t --abi armeabi-v7a -c 100M
+          script: ./gradlew connectedAndroidTest
+
+      - name: Test
+        uses: reactivecircus/android-emulator-runner@v2
+        with:
+          api-level: 22
+          emulator-options: --force -n test -t --abi armeabi-v7a -c 100M
+          script: ./gradlew testReleaseUnitTest
```

Comment:
So does this now need to use the emulator runner?

Reply:
Probably not